data/analytics/attributes.yml

```diff
@@ -75,13 +75,6 @@
   type: string
   redact: false
 
-- name: index
```

Comment:
I may have misunderstood this but would it be good to keep `index`? As it will exist as an attribute - maybe we could amend the description to mention that it will contain the `index_link`, `index_section` and `index_section_count` properties

Reply:
I think `index` only serves as a wrapper for other attributes, and therefore there isn't anything to document about it. Also we have the `event_data` attribute for all events, which is also only a wrapper for other attributes, which we don't document.

Reply:
Okey doke, fair enough.